tests/integration/test_vulnerability_detector/test_windows/test_cpe_indexing.py

```diff
@@ -0,0 +1,176 @@
+    {"target": "WINDOWS_XP", "os_name": "Microsoft Windows XP",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_xp",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_VISTA", "os_name": "Microsoft Windows Vista",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_vista",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_7", "os_name": "Microsoft Windows 7",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_7",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_8", "os_name": "Microsoft Windows 8",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_8",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_8.1", "os_name": "Microsoft Windows 8.1",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_8.1",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_10", "os_name": "Microsoft Windows 10",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_10",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_SERVER_2008", "os_name": "Microsoft Windows Server 2008",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_server_2008",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_SERVER_2008_R2", "os_name": "Microsoft Windows Server 2008 R2",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_server_2008",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_SERVER_2012", "os_name": "Microsoft Windows Server 2012",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_server_2012",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_SERVER_2012_R2", "os_name": "Microsoft Windows Server 2012 R2",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_server_2012",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_SERVER_2016", "os_name": "Microsoft Windows Server 2016",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_server_2016",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_SERVER_2019", "os_name": "Microsoft Windows Server 2019",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_server_2019",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_SERVER_2013_I386", "os_name": "Microsoft Windows Server 2003",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_server_2003",
+        "os_release": "1000", "architecture": "i386"},
```

Comment:
```suggestion
        "os_release": "1000", "architecture": "x86"},
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/804/commits/fe8a3fe80a24bbf13433b3d8a2c168742e2a1eb1